### PR TITLE
Fix/bugs 3

### PR DESCRIPTION
**Pull request by cjharkins**

### CODE REVIEW

File: packages/frontend/src/components/forms/AddOrEditDoctor.vue [hunk 1]
```diff
@@ -22,7 +22,7 @@
           :class="{ _invalid: invalidInput.lastName }"
         />
       </div>
-      <div class="mt3">
+      <div v-if="!add" class="mt3">
```
Comment (embryCODE) [inline packages/frontend/src/components/forms/AddOrEditDoctor.vue]: She mentioned something about only she should be able to edit phone numbers. Are you doing that or are you just disabling editing for everyone?

File: packages/frontend/src/components/forms/AddOrEditDoctor.vue [hunk 2]
```diff
@@ -131,7 +131,7 @@ export default {
         this.add &&
         !this.invalidInput.firstName &&
         !this.invalidInput.lastName &&
-        !this.invalidInput.phoneNumber &&
+        // !this.invalidInput.phoneNumber &&
```
Comment (embryCODE) [inline packages/frontend/src/components/forms/AddOrEditDoctor.vue]: If we don't want this line, remove it, don't just comment it out.

File: packages/frontend/src/components/forms/AddOrEditDoctor.vue (hunk 1 above)
Comment (cjharkins) [reply packages/frontend/src/components/forms/AddOrEditDoctor.vue]: You're right, I read it wrong. Only displaying edit options if admin on sign out sheet.

File: packages/frontend/src/components/forms/AddOrEditDoctor.vue (hunk 2 above)
Comment (cjharkins) [reply packages/frontend/src/components/forms/AddOrEditDoctor.vue]: uncommented line to keep validation consistent

**Pull request merged**